pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -151,6 +151,8 @@ public class ModularLoadManagerImpl implements ModularLoadManager {
     // Strategy used to determine where new topics should be placed.
     private ModularLoadManagerStrategy placementStrategy;
 
+    private ModularLoadManagerStrategy roundRobinBrokerSelector;
```

Comment:
nit: can we rename this to sheddingExcludedNamespaceSelectionStrategy in case we use another selection strategy in the future?